web/source/osk/defaultLayouts.ts

```diff
@@ -101,12 +101,6 @@ namespace com.keyman.osk {
       'leftctrl-shift': '*LCtrlShift*',
       'rightctrl-shift': '*RCtrlShift*'
     };
-
-    static _BaseLayout: string = 'us';       // default BaseLayout
-    static _BaseLayoutEuro: {[code: string]: string} = {
-      'se': '\u00a71234567890+´~~~QWERTYUIOP\u00c5\u00a8\'~~~ASDFGHJKL\u00d6\u00c4~~~~~<ZXCVBNM,.-~~~~~ ',  // Swedish
-      'uk': '`1234567890-=~~~QWERTYUIOP[]#~~~ASDFGHJKL;\'~~~~~\\ZXCVBNM,./~~~~~ ' // UK
```

Comment:
As it turns out, none of the OSK classes actually use either of these fields; they were only used for text processing.

Actually, that's not entirely true.  The second one (`_BaseLayoutEuro`) was completely unused!  If we do want to keep this around, it should probably be preserved on `keyMapping.ts` in some fashion - that's far more related than its original location.

Reply:
This was used in the past -- for physical keyboard mnemonic remapping. There was a Swedish organisation that used KeymanWeb a few years back and used mnemonic layouts along with this. My guess is that this usage got lost somewhere in KeymanWeb 10-12.